In [1]:
from scipy.stats import norm

#### Calculate the probability of a person that is 65 in is a Male or a Female:

- Male Normal Distribution Curve: N(μ=70, σ=4)

- Female Normal Distribution Curve: N(μ=65, σ=3.5)

#### P(x=65 | y=Male)

In [2]:
p_65_m = norm.pdf(x=65, loc=70, scale=4)
p_65_m

0.04566227134725548

#### P(x=65 | y=Female)

In [3]:
p_65_f = norm.pdf(x=65, loc=65, scale=3.5)
p_65_f

0.11398350868612363

#### P(y=Male) = P(y=Female) = 0.5

In [4]:
p_m_f = 0.5

#### P(x=65) = P(x=65 | y=Male)*P(y=Male) + P(x=65 | y=Female)*P(y=Female)

In [5]:
p_65 = p_65_m*p_m_f + p_65_f*p_m_f
p_65

0.07982289001668955

#### P(y=Male | x=65) = P(x=65 | y=Male)*P(y=Male)/P(x=65)

In [6]:
p_m_65 = p_65_m * p_m_f / p_65
p_m_65

0.2860224137318776

#### P(y=Female | x=65) = P(x=65 | y=Female)*P(y=Female)/P(x=65)

In [7]:
p_f_65 = p_65_f * p_m_f / p_65
p_f_65

0.7139775862681225

Therefore, since the probabilitiy of a person being female with 65 in is 71.4% is greater than the probability of a person being male with 65 in of 28.6%, than we may conclude that the person is a female.